src/lib.rs

```diff
@@ -277,11 +277,15 @@ extern crate serde_derive;
 extern crate unwrap;
 
 extern crate bincode;
+#[cfg(test)]
+extern crate maidsafe_utilities;
```

Comment:
If it's only for tests is it not better to put it as a dev dependency in Cargo.toml ?

Reply:
It is a dev dependency, see https://github.com/povilasb/p2p/blob/more-tests/Cargo.toml#L32 :)